### ISSUE TITLE
Use a single color for all icons

### ISSUE THREAD

**Issue by ccordoba12** [spyder-ide/ux-improvements#35]
This is something that came up during our discussions before, but unfortunately we didn't have time to address it for Spyder 5.

However, it's an interesting idea, so I'm writing it down here so we don't forget to reach an agreement about it for Spyder 6.

**Comment by conradolandia** [comment]
I think it's important to have an unified color palette for all the icons. This gives it consistency. Now, it could be interesting to make the icons themable; however, this would imply other changes. I don't know how complex would be, but we could think about theming the whole UI, so users can choose the color theme beyond Dark/Light and making the icon theming a subcategory of that particular issue. I could work on creating more color themes based on the current CSS theme.

**Comment by ccordoba12** [comment]
> I think it's important to have an unified color palette for all the icons.

Agreed. So you're not in favor of using a single color for all icons, right? That's what at some point we thought to do, but I was mainly the one against that idea.

> I don't know how complex would be, but we could think about theming the whole UI, so users can choose the color theme beyond Dark/Light and making the icon theming a subcategory of that particular issue

That requires some improvements in [QDarkstyle](https://github.com/ColinDuquesnoy/QDarkStyleSheet), the package we use to theme Spyder.

> I could work on creating more color themes based on the current CSS theme.

That would be great, thanks @conradolandia! I think the idea in that case would be to remove the interface theme option and instead make the entire interface follow the current syntax highlighting theme selected by users. That's very similar to how VSCode works regarding themes.

**Comment by CAM-Gerlach** [comment]
> Agreed. So you're not in favor of using a single color for all icons, right? That's what at some point we thought to do, but I was mainly the one against that idea.

It sounds like to me that @conradolandia was saying to just have one (or maybe a few) consistent accent colors between the icons, and allowing users to change that accent color, but I might be misunderstanding him here.

**Comment by conradolandia** [comment]
> It sounds like to me that @conradolandia was saying to just have one (or maybe a few) consistent accent colors between the icons, and allowing users to change that accent color, but I might be misunderstanding him here.

On principle yes, but I want to keep open the idea of giving full control of the overall color theme to the user, with our current theme and accent colors as sensible defaults. Not just a Dark/Light theme, but a selection of themes, with both dark and light variations. I could work on that after we finish with higher priority stuff.
